docs/philosophy/outcomes/developer_adoption.md

```diff
@@ -269,72 +210,72 @@ The simplest way to calculate _operational efficiency_ is by comparing the numbe
 
 #### Data Points
 
-_Adoption events ($E$)_
-
-:   A unique target user does something valuable using the tool
-
-    Expressed as a set of timestamps
+[_Adoption events $(E)$_](#adoption-events)
 
 _Component Maintainers ($M$)_
 
-:   A count of the number of people it takes to maintain the component
+:   The number of people it takes to maintain the component
 
     Expressed as a number
 
+    TODO this is constant or can increase/decrease over time?
```

Comment:
This is something that changes over time (for example, I have the number of maintainers for today, yesterday, etc) or I do not care about time here, just to have an accurate number at the present time?

Reply:
yes change over time as we have to hire into a team or as people leave the team